lightningd/invoice.c

```diff
@@ -700,7 +700,6 @@ static struct command_result *json_invoice(struct command *cmd,
 		   p_opt("fallbacks", param_array, &fallbacks),
 		   p_opt("preimage", param_tok, &preimagetok),
 		   p_opt("exposeprivatechannels", param_bool, &exposeprivate),
-		   p_opt("description_hash", param_sha256, &description_hash),
```

Comment:
Why removing this parameter ? It might be useful. Anyway if you want to remove it you should drop the first commit instead.

Reply:
According to BOLT11 if there's a description_hash in the invoice there shouldn't be a description, and vice-versa. The current API requires a description parameter, so I'm not sure how could we allow a description_hash to be passed.